packages/documentation/src/content/docs/overview/concepts/accounting.mdx

```diff
@@ -1,139 +1,278 @@
+<Steps>
+  1. Rafiki receives packets from a peer. These packets are all denominated in
+  EUR, worth €10. Your EUR liquidity increases to 20 (10 + 10). 2. The current
+  EUR-to-USD exchange rate is applied, with €10 equating to $12 USD. 3. Rafiki
+  moves $12 USD from your USD asset liquidity account into an incoming payment
+  liquidity account. Your USD liquidity decreases to 38 (50 - 12).
+</Steps>
 
-In a cross currency transaction:
+**Cross-currency transaction #2:**
 
-- Rafiki receives packets worth 10 EUR and sends packets worth 11 USD. Your EUR liquidity increases to 20 (10 + 10) and your USD liquidity decreases to 39 (50 - 11).
-- Rafiki receives packets worth 50 EUR which equates to 55 USD. This transaction fails because Rafiki does not have enough USD liquidity.
+<Steps>
+1. Rafiki receives packets from a peer. These packets are all denominated in EUR, worth €50. Your EUR liquidity increases to 70 (20 + 50).
+2. The current EUR-to-USD exchange rate is applied, with €50 equating to $55 USD.
+3. The transaction fails. Your USD liquidity account is 38, so you don't have enough liquidity to cover the transaction.
+4. Your EUR liquidity reduces back to 20 (70 - 50).
+</Steps>
+</details>
 
-### Peer liquidity
+#### Peer liquidity accounts
 
-Peer liquidity is the credit line, denominated in the asset of the peering relationship, you extend to a peer. Peer liquidity should be defined in the peering agreement and depends on the trust between you and your peer. If peer liquidity is insufficient, payments will not be processed. When peer liquidity is used up, you and your peer should settle then reset your liquidity.
+Peer liquidity is the credit line you've extended to a peer. A peer liquidity account represents the amount of the line of credit that the peer still has available to them. You have one liquidity account for each peer and the account is denominated in the asset you both agreed to transact in.
+
+The amount of credit that you extend to a peer, the asset that you transact in, and the mechanism you use to settle are just a few items that should be defined in your respective peering agreements.
 
 :::note
-A peering agreement is a legal contract between the parties involved in a peering relationship. It defines terms such as the assets involved, auth tokens, connection endpoints, and other operational details. It is not configured or managed within Rafiki but is necessary for establishing the terms under which assets are exchanged between peers.
+A peering agreement is a legal contract between the parties involved in a peering relationship. It defines terms such as the assets involved, auth tokens, connection endpoints, and other operational details. It is not configured or managed within Rafiki but is necessary for establishing the terms under which assets are exchanged.
```

Comment:
The auth tokens, connection endpoints are managed by Rafiki, so we can remove that part from the note